core/driver/sources/src/main/resources/reference.conf

```diff
@@ -161,34 +201,108 @@ probe {
       "https://cache-redirector.jetbrains.com/intellij-jbr/jbr-[major]-[platform]-x64-b[minor].tar.gz"
     ]
 
+    // The maximal number of retries that should be performed by resolvers when resolving a pattern to a resource.
```

Comment:
The retries are only relevant for **network requests** right?

Reply:
In the standard ide-probe processing logic - yes. However, it is possible that `retries` will be used for a non-network request resolving. It is also used (for some reason, which I do not understand) for **jar** URIs.
<img width="841" alt="image" src="https://user-images.githubusercontent.com/49597713/197960699-c59cf2c7-d03c-4d50-a426-2ee11fba40a2.png">

Reply:
But I'm not sure if a retry would make sense for such **jar** scenario to be honest. @lukaszwawrzyk - can you help us understand this part?

Reply:
Huh kinda makes sense for the rare case of network file systems (so a `file:///`-like URI, which still requires an access over network)... maybe doesn't harm to leave as is indeed